iotlabmqtt/asyncconnection.py

```diff
@@ -56,7 +56,7 @@ def start(self):
         self.create_socket(socket.AF_INET, socket.SOCK_STREAM)
         try:
             self.connect(self.address)
-        except:  # pylint:disable=broad-except,bare-except
+        except:  # noqa: E722 pylint:disable=broad-except,bare-except
```

Comment:
Could this be simplified like this:
```python
except:  # noqa # pylint: disable=broad-except,bare-except
```

Reply:
Yes, however that will disable flake8 for any type of warning for the line, not only the warning of the bare-except.